src/components/geocoder/Geocoder.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+
+  <v-toolbar-items>
+    <v-combobox
+      return-object
+      :no-filter="noFilter"
+      v-model="selected"
+      :autofocus="autofocus"
+      :items="resultItems"
+      :label="placeHolder"
+      append-icon=""
+      :dark="dark"
```

Comment:
By adding `:color="dark ? 'white': ''"` we could avoid the blue combobox, when having the focus. This would fit better to the general layout.

Reply:
Good! Could not find how to style/get rid of the blue CB!